### ISSUE TITLE
literature sources for production metadata &/ TsDC

### ISSUE THREAD

**Issue by moedn** [OPEN-NEXT/LOSH#84]
- PCB
  - https://www.linkedin.com/pulse/pcb-design-output-data-manufacturer-requires-amit-bahl/?articleId=6679094026790162432

**Comment by moedn** [comment]
or maybe just run a survey for maker spaces like

"Which sort of hardware can you manufacture?" (PCB, 3D printed, machined, welded,…)

"What metadata would you need in order to assess whether or not you can manufacture…

a PCB
[list of already suggested fields so people can upvote] 

a 3D printed part
[list of already suggested fields so people can upvote]

a machined part
[list of already suggested fields so people can upvote]

a welded assembly
[list of already suggested fields so people can upvote]

…

**Comment by nielek2** [comment]
Data hackathon?

**Comment by moedn** [comment]
@nielek2 yes (sorry for the late reply) I'd gather 'metadata suggestions from makers'. Since WP2 won't cover that, but it has been requested by the user stories and people generally seem reactive to that, we need another justification method how we came to the provided `production metadata`. For this first draft I called a few people skilled in mechanics, PCBs etc. and asked them what kind of information they'd need as metadata in order to compare and assess the producibility of a piece of hardware. but none of these discussions have been recorded.
--
secondly we'll run hackthons for a manually curated set of metadata. So: found cool OSH? (I have a list now) → let's create some high-quality metadata for them so they can be actually found (and compared etc.) on LOSH :tada: 

The data that we're getting from automatic crawls is rather messy. So in order to demonstrate the potential of what would be possible with good metadata, we build a small set of manually curated data for selected OSH. Maybe this even helps convincing platform to rething their metadata :)
